space-plugins/nuxt-starter/pages/index.vue

```diff
@@ -1,37 +1,28 @@
 <script setup lang="ts">
+//TODO: check if user is admin
 const { data } = await useFetch('/api/stories');
 </script>
 
 <template>
-	<div>
-		<div class="mx-20 my-11">
-			<Header title="Story Starter">
-				<template #icon>
-					<LucideTornado class="text-primary" />
-				</template>
-				<template #description>
-					This is a `space-plugin-nuxt-starter`.
-				</template>
-				<!-- <template #right-action-bar>
-					<button class="btn btn-outline">
-						<span class="text-base">Settings</span><LucideSettings :size="16" />
-					</button>
-				</template> -->
-			</Header>
+	<NuxtLink
+		class="btn btn-outline"
+		to="/settings?space_id=149096&user_id=110847"
```

Comment:
The query paramaters are necessary for the moment to retrieve the correct app session